src/utils/client.ts

```diff
@@ -27,32 +30,40 @@ const retryStrategy: IRetryStrategyOptions = {
     || (isAxiosError(error) && !!error.response?.status.toString().startsWith("5")),
 };
 
-export const createClient = ({ environmentId, apiKey, commandName }: Params): ManagementClient =>
+export const createClient = ({ environmentId, apiKey, commandName, baseUrl }: Params): ManagementClient =>
   // eslint-disable-next-line no-restricted-syntax
   new ManagementClient({
     environmentId,
     apiKey,
     retryStrategy,
     headers: [{ header: sourceTrackingHeaderName, value: `${packageJson.name};${packageJson.version};${commandName}` }],
     httpService: defaultHttpService,
+    baseUrl: resolveUrlParam(baseUrl, createManagementApiUrl),
   });
 
-export const createClientDelivery = (
-  { environmentId, previewApiKey, usePreviewMode, commandName }: DeliveryParams,
-): DeliveryClient =>
+export const createClientDelivery = (params: DeliveryParams): DeliveryClient =>
```

Comment:
so what's it going to be? destructuring or not? now `createClient` uses destructuring but `createClientDelivery` does not :D

Reply:
I'll unify it 😄